src/model/color.js

```diff
@@ -1,12 +1,5 @@
 export default class Color {
-    constructor({
-        name,
-        red,
-        green,
-        blue,
-        hex,
-        alpha
-    }) {
+    constructor({ name, red, green, blue, hex, alpha }) {
```

Comment:
Is there a specific reason why you changed the formatting? If not, could you revert this to keep things consistent?

Reply:
I’ll revert, I guess my formatter messed up